pkg/apis/kubeone/types.go

```diff
@@ -142,46 +160,63 @@ const (
 type HostConfig struct {
 	// ID automatically assigned at runtime.
 	ID int `json:"-"`
+
 	// PublicAddress is externally accessible IP address from public internet.
 	PublicAddress string `json:"publicAddress"`
+
 	// PrivateAddress is internal RFC-1918 IP address.
 	PrivateAddress string `json:"privateAddress"`
+
 	// SSHPort is port to connect ssh to.
 	// Default value is 22.
 	SSHPort int `json:"sshPort,omitempty"`
+
 	// SSHUsername is system login name.
 	// Default value is "root".
 	SSHUsername string `json:"sshUsername,omitempty"`
+
 	// SSHPrivateKeyFile is path to the file with PRIVATE AND CLEANTEXT ssh key.
 	// Default value is "".
 	SSHPrivateKeyFile string `json:"sshPrivateKeyFile,omitempty"`
+
 	// SSHAgentSocket path (or reference to the environment) to the SSH agent unix domain socket.
 	// Default value is "env:SSH_AUTH_SOCK".
 	SSHAgentSocket string `json:"sshAgentSocket,omitempty"`
+
 	// Bastion is an IP or hostname of the bastion (or jump) host to connect to.
 	// Default value is "".
 	Bastion string `json:"bastion,omitempty"`
+
 	// BastionPort is SSH port to use when connecting to the bastion if it's configured in .Bastion.
 	// Default value is 22.
 	BastionPort int `json:"bastionPort,omitempty"`
+
 	// BastionUser is system login name to use when connecting to bastion host.
 	// Default value is "root".
 	BastionUser string `json:"bastionUser,omitempty"`
+
 	// Hostname is the hostname(1) of the host.
 	// Default value is populated at the runtime via running `hostname -f` command over ssh.
 	Hostname string `json:"hostname,omitempty"`
+
 	// IsLeader indicates this host as a session leader.
 	// Default value is populated at the runtime.
 	IsLeader bool `json:"isLeader,omitempty"`
+
 	// Taints are taints applied to nodes. Those taints are only applied when the node is being provisioned.
 	// If not provided (i.e. nil) for control plane nodes, it defaults to:
 	//   * For Kubernetes 1.23 and older: TaintEffectNoSchedule with key node-role.kubernetes.io/master
 	//   * For Kubernetes 1.24 and newer: TaintEffectNoSchedule with keys
 	//     node-role.kubernetes.io/control-plane and node-role.kubernetes.io/master
 	// Explicitly empty (i.e. []corev1.Taint{}) means no taints will be applied (this is default for worker nodes).
 	Taints []corev1.Taint `json:"taints,omitempty"`
+
+	// Lables to be used to apply (or remove, with minus sign suffix, see more kubectl help label) lables to/from node
+	Labels map[string]string `json:"labels"`
```

Comment:
Why don't we have omitempty here? We have it for other fields.

Reply:
Hm, probably forgotten :shrug: